src/status_im/utils/varint.cljs

```diff
@@ -1,7 +1,7 @@
 (ns ^{:doc "Implementation of varint based on https://github.com/chrisdickinson/varint"}
  status-im.utils.varint
-  (:require [status-im.ethereum.abi-spec :as abi-spec]
-            ["web3-utils" :as utils]))
+  (:require ["web3-utils" :as utils]
```

Comment:
hope we can remove `web3-utils` dependency now :)

Reply:
Hi @flexsurfer ,  i found we used following functions with web3-utils in other places:
- `hexToUtf8`
- `sha3`
- `utf8ToHex`
- `isAddress`
- `toChecksumAddress`
- `checkAddressChecksum`

Can we make a separate issue for removing dependency `web3-utils` ?

Reply:
yes would be great

Reply:
could u create the issue? @flexsurfer 

> yes would be great

Reply:
https://github.com/status-im/status-mobile/issues/13888